docs/sources/flow/reference/components/discovery.serverset.md

```diff
@@ -0,0 +1,93 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.serverset/
+title: discovery.serverset
+---
+
+# discovery.serverset
+
+`discovery.serverset` discovers [Serversets][] stored in Zookeeper and exposes them as targets.
+Serversets are commonly used by [Finagle][] and [Aurora][].
+
+[Serversets]: https://github.com/twitter/finagle/tree/develop/finagle-serversets
+[Finagle]: https://twitter.github.io/finagle/
+[Aurora]: https://aurora.apache.org/
+
+## Usage
+
+```river
+discovery.serverset "LABEL" {
+	servers = SERVERS_LIST
+	paths   = ZK_PATHS_LIST
```

Comment:
I don't know if ZK is a well-known prefix but I found it confusing for a second; probably better to spell it out:

```suggestion
	paths   = ZOOKEEPER_PATHS_LIST
```